astropy/coordinates/angle_utilities.py

```diff
@@ -28,19 +28,7 @@
                      IllegalSecondWarning, IllegalSecondError)
 from astropy.utils import format_exception
 from astropy import units as u
-
-TAB_HEADER = """# -*- coding: utf-8 -*-
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-
-# This file was automatically generated from ply. To re-generate this file,
-# remove it from this folder, then build astropy and run the tests in-place:
-#
-#   python setup.py build_ext --inplace
-#   pytest astropy/coordinates
-#
-# You can then commit the changes to this file.
-
-"""
+import astropy.utils.parsing as parsing
```

Comment:
Nitpick: This is more consistent with the style of the existing imports.
```suggestion
from astropy.utils import parsing
```

Reply:
Done (I merged it into the same line as another `from astropy.utils import ...`).